frontend/src/features/mapSideBar/tabs/propertyResearch/PropertyResearchTabView.tsx

```diff
@@ -0,0 +1,67 @@
+import { SectionField } from 'features/mapSideBar/tabs/SectionField';
+import { StyledFormSection, StyledSectionHeader } from 'features/mapSideBar/tabs/SectionStyles';
+import { Api_ResearchFileProperty } from 'models/api/ResearchFile';
+import * as React from 'react';
+import styled from 'styled-components';
+
+interface PropertyResearchFile {
+  id: number;
+  descriptiveName: string;
+  purpose: string;
+  legalOpinionRequired: string;
+  legalOpinionObtained: string;
+  documentReference: string;
+  sumaryNotes: string;
+}
+
+export interface IPropertyResearchTabViewProps {
+  researchFile: Api_ResearchFileProperty;
+}
+
+const PropertyResearchTabView: React.FunctionComponent<IPropertyResearchTabViewProps> = props => {
+  const detail: PropertyResearchFile = {
+    id: props.researchFile.id || 0,
+    descriptiveName: props.researchFile.propertyName || '',
+    purpose:
+      props.researchFile.propertyPurpose?.map(x => x.propertyPurposeType?.description).join(', ') ||
+      '',
+    legalOpinionRequired:
+      props.researchFile.isLegalOpinionRequired !== undefined
+        ? props.researchFile.isLegalOpinionRequired
+          ? 'Yes'
+          : 'No'
+        : '',
+    legalOpinionObtained:
```

Comment:
can we get a util function for this? we may already have one.

Reply:
We could, will take a look